code/modules/urist/_helpers.dm

```diff
@@ -384,3 +384,21 @@
 		if(light_amt < max_light)
 			return 1
 	return 0
+
+//for throwing things from one turf to another
+
+/proc/launch_atom(var/atom/movable/projectile, var/turf/start_turf, var/turf/target_turf)
+	if(ispath(projectile))
+		new projectile(start_turf)
```

Comment:
I'm not 100%, but does this actually shadow `projectile` automatically with the new spawned atom? I would've thought not?

Reply:
already fixed locally, i'm just testing things with the pdc right now. good catch though, i had a minute where i was very confused why the Nerva wasn't getting blasted